.editorconfig

```diff
@@ -227,6 +227,8 @@ dotnet_naming_style.I_style.capitalization = pascal_case
 # Rules
 # ----------------------------------------------------------------------------------------------------------------------
 
+dotnet_diagnostic.CA1822.severity = warning # Increase visibility for Member 'xxx' does not access instance data and can be marked as static
```

Comment:
See the last commit. I tend to miss them often and this will increase the visibility of the suggestion.